controllers/TuitController.ts

```diff
@@ -256,7 +251,7 @@ export default class TuitController implements TuitControllerI {
                         "count": 1
                     }
                     // If the almost tag is a new one
```

Comment:
what if the tuit update includes a pre-existing tag in which case we would just need to increment the tag's count? do we account for that somewhere else that I'm missing?

Reply:
We need to add an else on 266 to increment the count if the tag already exists.  Good thinking.

Reply:
createTag in the tagDao should account for this:

    createTag = async (tag: Tag): Promise<Tag> => {
        const newTag = tag.tag; // The potentially new tag

        // Acquire tag if it exists
        let existingTag = await this.findTagByText(newTag.toString());
        // If tag already exists
        if (existingTag) {
            // Then increase count by one and update that tag
            existingTag.count++;
            await this.updateTag(existingTag);
            return this.findTagByText(existingTag.tag);
        }
        // Else
        else {
            // create a new tag
            return TagModel.create(tag);
            }
    }